LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -16,68 +16,18 @@ class PlaygroundEdvard : ISimulation
         List<Guid> subjectIdList = new List<Guid>();
         List<Guid> moduleIdList = new List<Guid>();
 
-        public static void Action0()
-        {
-            ObjectGenerator.GenerateAddress();
-        }
-        
-        public static void Action1()
-        {
-            Address address0 = new Address("dasdasd str",23, 15, "Yerevan", "asdqwe", "15", "Armenia");
-            ContactInfo contact0 = new ContactInfo
-            (
-                address0,
-                "zxczxc.asdasdqweqweqwe@gmail.com",
-                "010-54-74-90",
-                "010-28-88-88",
-                "013-23-122-33"
-            );
-
-            Employee employee0 = new Employee();
-            Employee employee1 = new Employee();
-
-            List<Employee> employeeList = new List<Employee>
-            {
-                employee0,
-                employee1
-            };
-
-            Department department0 = new Department(Guid.NewGuid(), contact0, "Dep0", "Department0", employeeList);
-
-            List<Department> departmentList = new List<Department>
-            {
-                department0
-            };
-
-            Unit<Department> unit0 = new Unit<Department>(UnitType.Department, "Dep0", "Department0", departmentList);
-
-            Console.WriteLine(AttributeGenerator.GetDateOfBirth(20, 60));
-
-            //unit0.Report();
-        }
-
-        public void Action2(int accountCount)
-        {
-            CreateAccounts(accountCount);
-            ActivateAccounts();
-        }
-
-        public void Action3(int subjectCount, int moduleCount)
-        {
-            CreateSubjects(subjectCount);
-            CreateModules(moduleCount);
-            AssignModulesToStudents();
-            AssignModulesToInstructors();
-        }
-
         public void CreateAccounts(int accountCount)
         {
+            ReportHeader("Create Account");
+
             for (int i = 1; i <= accountCount; i++)
             {
-                Console.WriteLine($" Account {i}");
+                ReportItem("Account", i);
```

Comment:
I would like to see more compact implementation of this part:
ReportItem("Account", i);
Account account = ObjectGenerator.GenerateAccount();
account.Report();
Something like:
Account account = ObjectGenerator.GenerateAccount();
ReportItem(account.ToString()), "Activated account",  i);
You can change ReportItem to take in three parameters:
ReportItem(string itemDescription, string actionDescription, int itemIndex)